packages/sdk/src/services/config.service.ts

```diff
@@ -1,32 +1,59 @@
-import {BigNumberish} from 'ethers'
 import {Environment, getEnvironment} from '@holographxyz/environment'
 import {getNetworkByChainId, Network} from '@holographxyz/networks'
+
 import {HolographLogger} from './logger.service'
 import {UnavailableNetworkError, UnknownError, normalizeException} from '../errors'
+import {getEnv} from '../config/env.validation'
+import {CHAIN_ID_BY_RPC_URL} from '../constants/rpcs'
+import {isFrontEnd} from '../utils/helpers'
 
 export type ChainsRpc = Record<number, string>
 
+export type HolographConfig = {
+  networks?: ChainsRpc
```

Comment:
Can we make the ChainRpc type more specific? So we only allow specific chainIds and not any Record<number, string> combo. For example, ij should not be able to set chainId = 5 because we are no longer on georli.

Reply:
Done ✅ 

I also implemented using the network name as the object key like:

networks = {
     ethereum: URL,
     polygonTestnet: URL
}